cmd/proxy/actions/dotfile.go

```diff
@@ -0,0 +1,30 @@
+package actions
+
+import (
+	"io/ioutil"
+	"log"
+	"path/filepath"
+
+	"github.com/mitchellh/go-homedir"
+)
+
+// initializeDotFile checks if provided dot file is at a pre-configured path
+// and moves to home directory -- note that this will override whatever
+// dot file you have in your home directory.
+func initializeDotFile(path string) {
+	if path == "" {
+		return
+	}
+
+	fileBts, err := ioutil.ReadFile(path)
+	if err != nil {
+		log.Fatalf("could not read %s: %v", path, err)
+	}
+
+	hdir, err := homedir.Dir()
+	if err != nil {
+		log.Fatalf("could not get homedir: %v", err)
+	}
+	rcp := filepath.Join(hdir, filepath.Base(path))
+	ioutil.WriteFile(rcp, fileBts, 0666)
```

Comment:
ignoring error we should at least log what's wrong. it will help a person troubleshooting the issue.
 also please mention in the doc that any .netrc/hgrc file in home will be replaced by what is in `path`

Reply:
I don't saw that missed error ( it was also missed previously), will add that.

About the documentation, we don't know in that function which file will be replaced, and this description like this is added in config.dev.toml. Should I maybe add it also to proxy.md I mean this part with override?

Reply:
we know that files copied using these methods will be replaced so i would start with that and add a warning to each of them. right at the place, you modified proxy.md